tests/test_integration.py

```diff
@@ -21,7 +19,10 @@ def test_large_select_queryset_with_writer(s3, large_select_queryset):
     """
     Test file writer and queryset with large result set.
     """
-    s3.meta.client.create_bucket(Bucket=BUCKET_NAME)
+    s3.meta.client.create_bucket(
+        Bucket=BUCKET_NAME,
+        CreateBucketConfiguration={"LocationConstraint": "eu-west-1"},
```

Comment:
Actual change.

This was done throughout. Without `LocationConstraint` all tests were failing.